Angular/my-app6-pipe/src/directive/checkmail.directive.ts

```diff
@@ -0,0 +1,24 @@
+import { Directive, ElementRef, Input, input, Output, TemplateRef, ViewContainerRef } from '@angular/core';
+
+@Directive({
+  selector: '[appCheckmail]',
+  standalone: true
+})
+export class CheckmailDirective {
+
+  constructor(private template : TemplateRef<string>, private view : ViewContainerRef) { }
```

Comment:
constructor after inputs and class  members

Reply:
done